universal/wrapper/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ function(pico_add_extra_outputs TARGET)
         get_target_property(BINARY_TYPE ${SOURCE_TARGET} PICO_TARGET_BINARY_TYPE)
         if (${BINARY_TYPE} STREQUAL "no_flash")
             message("Packaging no_flash universal binary to SRAM, so it has a load_map")
-            pico_package_uf2_output(${SOURCE_TARGET} 0x20000000)
+            pico_ensure_load_map(${SOURCE_TARGET})
```

Comment:
can we maybe clarify the message and/or add a comment ... it isn't clear what or why this is packaging

Reply:
Have clarified the message and added a comment explaining why